src/Sylius/Bundle/ReviewBundle/Doctrine/ORM/Subscriber/LoadMetadataSubscriber.php

```diff
@@ -106,11 +106,13 @@ private function createReviewerMapping($reviewerEntity, ClassMetadata $reviewerE
         return [
             'fieldName' => 'author',
             'targetEntity' => $reviewerEntity,
-            'joinColumn' => [
-                'name' => 'author_id',
-                'referencedColumnName' => $reviewerEntityMetadata->fieldMappings['id']['columnName'],
-                'nullable' => false,
-                'onDelete' => 'CASCADE',
+            'joinColumns' => [
```

Comment:
Ok, no worries. But, in that case, to be consistent, we should also update the `createSubjectMapping` method in that same file, as it has the same formatting (see https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ReviewBundle/Doctrine/ORM/Subscriber/LoadMetadataSubscriber.php#L87).

If you wish, I can update this PR to include that changes for both method.